### PR TITLE
Default template modifications

### PR DESCRIPTION
**Pull request by daniel-montalvo**
Simplifying text for Assumptions and Accessibility Support and removing dot at the end of the sentence for "Accessibility Support". Addressess #932

<< Describe the changes >>

Closes issue(s):

- (ADD ISSUE NUMBER HERE)

Need for Final Call:
<< choose one of the following and remove the rest >>
<< check https://act-rules.github.io/pages/design/process/#final-call-aka-call-for-consensus-cfc >>
This can be merged with 1 approval << choose reason: editorial changes to website/test code, adding new contributor, other (explain). >>
This will not require a Final Call << choose reason(s): editorial changes, changes to assumptions, background, accessibility support, change to website/test code (not rule), other (explain). >>
This will require a 1 week Final Call << small changes affecting a small number of test cases, if in doubt do not use this. >>
This will require a 2 weeks Final Call << new rule, or substantial changes affecting a large number of test cases, if in doubt, use this. >>

---

### **When creating PR:**

- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Comment by Jym77** [comment]
I'm not super-fan of making the stuff less "correct sentence". The removing of the final dot is particularly bothering me (I'd rather add the missing ones).

File: pages/design/composite-template-empty.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ For each test target, the outcome of (at least one of | all of | any of etc.) th
 
 ## Assumptions
 
-_There are currently no assumptions._
+_No assumptions_
 
 ## Accessibility Support
 
-_There are no major accessibility support issues known for this rule._
+_No accessibility support issues known._
```
Comment (kasperisager) [inline pages/design/composite-template-empty.md]: I'm guessing the period should removed here as well?

Comment (kasperisager) [reply pages/design/composite-template-empty.md]: Also, I do agree with @Jym77 that periods should be added, not removed: https://github.com/act-rules/act-rules.github.io/pull/935#issuecomment-534599986

**Comment by daniel-montalvo** [comment]
> Also, we need to update all the rules as a part of this PR, in my opinion.

We can broad the scope of this but it would take longer. 

> Also the pull request templates under the `.github` directory.

Will take a look at these.

**Comment by daniel-montalvo** [comment]
> Also the pull request templates under the `.github` directory.

Not sure what you mean. Under the .github directory there are two files:
* PULL_REQUEST_TEMPLATE.md
* ISSUE_TEMPLATE.md

I haven't found these strings in any of those files.

**Pull request merged**